nexus-repository-p2-it/src/test/it-resources/artifacts-mirror-removed.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?><?artifactRepository class='org.eclipse.equinox.internal.p2.artifact.repository.simple.SimpleArtifactRepository' version='1.0.0'?><repository name="HelloWorld Artifacts" type="org.eclipse.equinox.p2.artifact.repository.simpleRepository" version="1.0.0">
```

Comment:
what this file for?

Reply:
I am unsure myself. I believe I took most of the structure from the Conda Community Format. I will check in with @DarthHater and @bhamail

Reply:
Pretty sure this is for a Unit Test that we repurposed for a IT (because what we tested is more of an integration test, but we wrote a pretty complicated Unit Test in the past to make up for there being no IT framework).